docs/contributing/development-policies.rst

```diff
@@ -47,28 +47,50 @@ the author of the pull request.
 Proposal and discussion of significant changes
 **********************************************
 
-New features and backward-incompatible changes should be proposed using the `Discourse forum <https://discourse.django-cms.org>`_. Discussion should take place there before any pull requests
-are made.
+New features and backward-incompatible changes should follow the best practice of DEPS and
```

Comment:
Maybe we can include link to DEPS, because I don't know what is it =)

Reply:
Is it Django Enhancement Proposals?

Reply:
yes https://github.com/django/deps

Reply:
Good catch Monsieur!